OperatorOverloading/OperatorOverloading.Dbl/Convert.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Net;
+using System.Net.Sockets;
+using System.IO;
+
+
+namespace OperatorOverloading.Dbl
+{
+    public class Convert : IExchangeRateProvider
+    {
+
+        public double ConversionRate(string from, string to)
+        {
+            double convertFactor;
+            string jsonData = null; ;
+
+            jsonData = System.IO.File.ReadAllText(@"D:\Training\01 OperatorOverloading\OperatorOverloading.Model\Data.txt");
+            string searchString = from.ToUpper() + to.ToUpper();
+            convertFactor = Parse(jsonData, searchString);
+            if (convertFactor < 0)
+            {
+                throw new Exception("no results");
+            }
+            return convertFactor;
+        }
+
+        public double Parse(string jsonString, string searchString)
+        {
+            double rate = 0;
+            if (String.IsNullOrWhiteSpace(jsonString))
+            {
+                throw new Exception("InvalidResponse");
```

Comment:
Please change your exception message to make some sense. Also use resource file for the messages.